scripts/download_openmeteo_2024_2025.py

```
@@ -0,0 +1,151 @@
+#!/usr/bin/env python3
+import subprocess
+import sys
+import time
+from datetime import datetime, timedelta
+from pathlib import Path
+
+def download_openmeteo_with_rate_limit():
```

Ruff (0.12.2)

`download_openmeteo_with_rate_limit` is too complex (19 > 10)

(C901)